velox/common/memory/MallocAllocator.cpp

```diff
@@ -309,6 +324,10 @@ void* MallocAllocator::allocateBytesWithoutRetry(
     uint64_t bytes,
     uint16_t alignment) {
   if (!incrementUsage(bytes)) {
+    VELOX_MEM_LOG_EVERY_MS(WARNING, 1000)
+        << "Failed to allocateBytes " << succinctBytes(bytes)
+        << ": Exceeded memory allocator capacity limit of "
+        << succinctBytes(capacity_);
```

Comment:
Do we want to set allocation error message here?

Reply:
added and also added corresponding unit tests